source/4.0/learn/repositories/reading-simple-objects.html.md

```diff
@@ -70,6 +70,11 @@ rom = ROM.container(:sql, 'sqlite::memory') do |config|
     column :name, String, null: false
     column :email, String, null: false
   end
+  
+  config.relation(:users) do
+    schema(infer: true)
+    auto_struct true
```

Comment:
`auto_struct` is not needed here, repos configure it for their relations

Reply:
Whoops.. I knew that too.